### ISSUE TITLE
Emojis not working

### ISSUE THREAD

**Issue by infinite-persistence** [OdyseeTeam/odysee-frontend#1262]
Tried to select from the drop down, but the resulting comment is just text

<img width="390" alt="image" src="https://user-images.githubusercontent.com/64950861/161208410-4b28cd9b-56f1-45d4-9145-2081c4f82fa9.png">

**Comment by tzarebczan** [comment]
Looks like it's some emojis, but I think most work.

**Comment by saltrafael** [comment]
yeah some may be unsupported? would have to dig into the emoji library

**Comment by infinite-persistence** [comment]
- [ ] I wanted to just type `:heart:`, but it's way down below.  I would assume the ending colon would result in an exact match
